### PR TITLE
Mitid erhverv

### PR DESCRIPTION
**Pull request by sgryt**

### CODE REVIEW

File: src/pages/verify/e-ids/danish-mitid.mdx [hunk 1]
```diff
@@ -30,6 +32,42 @@ import ConsentRevocation from '../../../components/ConsentRevocation';
...
+  "sub": "{159d89fc-a2db-4300-a52a-b7865f7b1ff3}",
+  "uuid": "3c6d9757-1e70-438a-8dd3-5f84398c2e25",
+  "cvrNumberIdentifier": "12345678",
+  "2.5.4.10": "Testorganisation nr. 12345678",
+  "name": "Firstname Lastname",
+  "country": "DK"
+}
+```
+
+The `uuid` value here will not be the same as for a citizen-MitID login, even if the user chose to use their personal MitID for login.
```
Comment (mickhansen) [inline src/pages/verify/e-ids/danish-mitid.mdx:69]: Will this also affect the `sub` value then?

File: src/pages/verify/e-ids/danish-mitid.mdx [hunk 2]
```diff
@@ -30,6 +32,42 @@ import ConsentRevocation from '../../../components/ConsentRevocation';
...
+  "nameidentifier": "0f9960a0d28d4353a3e2ea07f8ffa185",
+  "sub": "{0f9960a0-d28d-4353-a3e2-ea07f8ffa185}",
+  "uuid": "74ffcd31-fbaf-4c33-bdac-169f25c1e416",
+  "cprNumberIdentifier": "2101270087",
+  "birthdate": "1927-01-21",
+  "age": "93",
+  "name": "Ditlev Von Testesen",
+  "cvrNumberIdentifier": "12345678",
+  "2.5.4.10": "Testorganisation nr. 12345678",
+  "companySignatory" : "true"
```
Comment (mickhansen) [inline src/pages/verify/e-ids/danish-mitid.mdx:49]: missing `,` before line break

File: src/pages/verify/e-ids/danish-mitid.mdx (hunk 1 above)
Comment (sgryt) [reply src/pages/verify/e-ids/danish-mitid.mdx:69]: Yes. Will elaborate.

File: src/pages/verify/e-ids/danish-mitid.mdx [hunk 3]
```diff
@@ -30,6 +32,42 @@ import ConsentRevocation from '../../../components/ConsentRevocation';
...
+  "country": "DK"
+}
+```
+
+### MitID for business ('MitID Erhverv')
```
Comment (mickhansen) [inline src/pages/verify/e-ids/danish-mitid.mdx:54]: postfix with something like `#jwttoken-examples-mitid-business` to counteract the duplicate anchor issue.

Comment (sgryt) [reply src/pages/verify/e-ids/danish-mitid.mdx:54]: addressing now

**Review by mickhansen — APPROVED** [review]
LGMT, can merge pending approval @flensted (or merge if not required).

**Comment by sgryt** [comment]
@flensted  keep-alive

**Pull request merged**